test/api-digester/Inputs/cake2.swift

```diff
@@ -141,3 +141,5 @@ public class EscapingFunctionType {
   public func removedEscaping(_ a: ()->()) {}
   public func addedEscaping(_ a: @escaping ()->()) {}
 }
+
+prefix operator ..*..
```

Comment:
Hm. What happens if we add the prefix operator but don't remove the infix operator? Is that correctly recorded as a source-compatible addition?

Reply:
no, we don't treat any additional decls as source/ABI breaking except stored property in non-resilient types and added requirements. Are added operators always source breaking?

Reply:
Sorry, they're not source-breaking. I guess I just want a test that it's not counted as source-breaking by accident.

Reply:
ah, no, it won't. Two infix operators will match with each other for comparison and the added prefix operator will be a source-compatible addition.